### PR TITLE
issues/522 - fix minimum buffer length check

### PR DESCRIPTION
**Pull request by cordellcalitz**
Check that request buffer is at least 6 bytes long to allow proper extraction of payload length.

### CODE REVIEW

File: test/Lint/test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const paths = [
 const options = {
     // Specify style of output
     formatter: "compact", // Defaults to `stylish`
-    timeout: 5000
+    timeout: 10000
```
Comment (cordellcalitz) [inline test/Lint/test.js:19]: lint timed out on my machine after 5s

**Comment by yaacov** [comment]
Thank you for the pull request :green_heart:

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "modbus-serial",
-  "version": "8.0.13",
+  "version": "8.0.14",
```
Comment (yaacov) [inline package.json:3]: The vertion bump is out of topic, can you undo this change ?

Comment (cordellcalitz) [reply package.json]: Sure, rolled back.

**Comment by cordellcalitz** [comment]
@yaacov Are you planning to publish this fix to npm?

**Comment by yaacov** [comment]
https://www.npmjs.com/package/modbus-serial/v/8.0.14 :tada: :cake:

**Comment by cordellcalitz** [comment]
Awesome, thanks!
